### ISSUE TITLE
Expression 'undefined' in attribute 'supported' used with directive 'clipboard' is non-assignable!

### ISSUE THREAD

**Issue by niemyjski** [omichelsen/angular-clipboard#31]
I run the [exceptionless project](https://github.com/exceptionless/Exceptionless) and I found this in my error logs this morning:

```
Error: [$compile:nonassign] Expression 'undefined' in attribute 'supported' used with directive 'clipboard' is non-assignable!
http://errors.angularjs.org/1.5.5/$compile/nonassign?p0=undefined&p1=supported&p2=clipboard
```

Brower:
```
Mozilla/5.0 (Windows NT 6.3; WOW64; rv:45.0) Gecko/20100101 Firefox/45.0
```

I believe it happened on this [line of code](https://github.com/exceptionless/Exceptionless.UI/blob/master/src/app/project/configure.tpl.html#L57). There is no supported attribute defined so I think the directive has some issues with this. I tried to reproduce locally on firefox on OSX but couldn't reproduce.

**Comment by omichelsen** [comment]
This should be fixed in v1.4.1 (see #29).

**Comment by niemyjski** [comment]
Thanks!
